CMSgov/mdct-mcr
# MDCT-1947: DynamicFields Bug

## Description

Closes [MDCT-1947](https://qmacbis.atlassian.net/browse/MDCT-1947). 

When adding a new row in `DynamicFields`, the previous values are pre-filling that newly added row (see the attached video on the ticket).

### How to test

Run locally, visit `http://localhost:3000/mcpar/program-information/add-plans` and go crazy with the rows! Refresh, add more, all should work as expected.

### Changed Dependencies

N/A

## Code author checklist

## Reviewer checklist (two different people)
